src/lighteval/tasks/requests.py

```diff
@@ -136,6 +136,7 @@ class GreedyUntilMultiTurnRequest(Request):
     stop_sequence: str
     generation_size: int
     request_type = RequestType.GREEDY_UNTIL_MULTI_TURN
+    use_logits: bool = False
```

Comment:
Not sure I see the link with the rest of the PR

Reply:
this was forgoten in another PR, it is needed to make multiturn tasks work